### PR TITLE
Add tests for workflow groups

### PR DESCRIPTION
**Pull request by pcambra**

### CODE REVIEW

**Comment by pcambra** [comment]
Just wondering how to test the valid yml file for process definitions, I don't think we should be asserting for a not exception being thrown

File: tests/src/Unit/WorkflowGroupManagerTest.php [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    $this->workflowGroupManager = new TestWorkflowGroupManager($this->moduleHandler, $this->cache);
+  }
+
+  /**
+   * @covers ::processDefinition
+   */
+  public function testProcessNoLabelDefinition() {
+    vfsStream::setup('root');
+
+    $file = <<<'EOS'
```
Comment (bojanz) [inline tests/src/Unit/WorkflowGroupManagerTest.php]: Can we do an array + Yaml encode? Would look cleaner.

Comment (pcambra) [inline tests/src/Unit/WorkflowGroupManagerTest.php]: Following core patterns but makes sense

**Comment by pcambra** [comment]
Added the assertions part

**Comment by pcambra** [comment]
Not completely sure if we're doing the right thing by testing the WorflowGroup methods here, I think they should belong to WorkflowGroupTest

**Comment by pcambra** [comment]
This might be good to go

**Pull request merged**